### PR TITLE
Fix Code Smells

### PR DESCRIPTION
**Pull request by johnpcooke94**
Fixing all of the code smells highlighted by SonarCloud. Very minor stuff for the most part, mostly just making properties readonly that are never changed after instantiation. Also did a minor refactor on a unit test.

Something to note - I did not fix the code smells for this issue highlighted by Jason:
image

Per his advice, this is a non-issue so I created a new quality profile that ignores this rule, so we should no longer see it highlighted.

### CODE REVIEW

File: CSLabs.ConsoleUtil/Commands/ListHypervisors.cs [hunk 1]
```diff
@@ -14,8 +14,10 @@ public class ListHypervisorsOptions
     
     public class ListHypervisorsCommand : AsyncCommand<ListHypervisorsOptions>
     {
-        private DefaultContext _context;
-        private string _encryptionKey;
+        private readonly DefaultContext _context;
+        
+        // Do we need this field? It's totally unread, only ever set in the constructor but never used
+        private readonly string _encryptionKey;
```
Comment (johnpcooke94) [inline CSLabs.ConsoleUtil/Commands/ListHypervisors.cs]: @jasekiw @Cliftonz can we delete this property? It's never read after instantiation, the only reference to it is in the constructor.

Comment (Cliftonz) [reply CSLabs.ConsoleUtil/Commands/ListHypervisors.cs]: That would be a jason question.

Comment (Cliftonz) [reply CSLabs.ConsoleUtil/Commands/ListHypervisors.cs]: I do know that we will be changing from passwords to apikeys soon when communicating to proxmox.

Comment (johnpcooke94) [reply CSLabs.ConsoleUtil/Commands/ListHypervisors.cs]: Yeah that was why I wanted to check with you guys, I wasn't sure if there were future plans to use this property or not, so I wanted to check in before I just removed it. Sonarcloud isn't happy with it being here, but I can mark that code smell as resolved if we have plans to use it soon.

Comment (jasekiw) [reply CSLabs.ConsoleUtil/Commands/ListHypervisors.cs]: It think this was accidentally copied from another command, it can be deleted.

**Pull request merged**